onnxruntime/core/providers/rknpu/onnx_converter.h

```diff
@@ -0,0 +1,239 @@
+  void AddLayerMul(const std::string& input1, const std::string& input2,
+                   const std::string& output);
+  void AddLayerAdd(const std::string& input, float scalar,
+                   const std::string& output);
+  void AddLayerMul(const std::string& input, float scalar,
+                   const std::string& output);
+  void AddLayerDequantize(const std::string& input,
+                          const std::string& output);
+  void AddLayerLRN(const std::string& input, int32_t radius, float bias,
+                   float alpha, float beta, const std::string& output);
+  void AddLayerTanh(const std::string& input, const std::string& output);
+  void AddLayerFloor(const std::string& input, const std::string& output);
+  void AddLayerLogistic(const std::string& input, const std::string& output);
+
+  void AddLayerBatchNorm(const std::string& input,
+                         const std::string& scale_name,
+                         const std::string& bias_name,
+                         const std::string& mean_name,
+                         const std::string& var_name,
+                         const float eps,
+                         const std::string& output);
+  void AddLayerReshape(const std::string& input, const std::string& shape_name,
+                       const std::string& output);
+  void AddLayerFlatten(const std::string& input, const int32_t axis,
+                       const std::string& output);
+  void AddLayerTranspose(const std::string& input,
+                         const std::vector<int32_t>& perm,
+                         const std::string& output);
+  void AddLayerSqueeze(const std::string& input,
+                       const std::vector<int32_t>& axes,
+                       const std::string& output);
+  void AddLayerUnsqueeze(const std::string& input,
+                         const std::vector<int32_t>& axes,
+                         const std::string& output);
+  void AddLayerGather(const std::string& input, const std::string& indices_name,
+                      const int32_t axis, const std::string& output);
+  void AddLayerLeakyRelu(const std::string& input, const float alpha,
+                         const std::string& output);
+  void AddLayerClip(const std::string& input,
+                    const int32_t min,
+                    const int32_t max,
+                    const std::string& output);
+  void AddLayerDequantizeLinear(const std::string& input,
+                                const std::string& input_scale,
+                                const std::string& input_zp,
+                                const std::string& output);
+  void AddLayerQuantizeLinear(const std::string& input,
+                              const std::string& output_scale,
+                              const std::string& output_zp,
+                              const std::string& output);
+
+  void Clear();
+
+ public:
+  OnnxConverter() {}
+  virtual ~OnnxConverter() { Clear(); }
```

Comment:
Why introducing a virtual table when a) not overriding anthing, b) the class is not designed for inheritance

Reply:
it has been removed.